### ISSUE TITLE
Code insiders: automatic tasks are not run after update

### ISSUE THREAD

**Issue by wclr** [microsoft/vscode#125243]
After insiders update, tasks that were run automatically on workspace load are not run anymore:

Here is task config .vscode/tasks.json:

```json
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "env-gen",
      "type": "shell",
      "command": "pnpx",
      "args": ["ts-node-dev", "--respawn", "-T", "./config/env-gen"],
      "isBackground": true,
      "presentation": {
        "echo": true,
        "reveal": "silent",
        "revealProblems": "always",
        "focus": false,
        "panel": "shared",
        "showReuseMessage": true,
        "clear": false
      },
      "runOptions": { "runOn": "folderOpen" }
    },
  ]
}
```

I run `Tasks: Manage automatic tasks in forlder` `Allow` but it doesn't help.

Have to run it manually to see it in the terminal. Before the update, it worked ok with this config.
What can be the problem is there a way to debug what is going on with automatic task running?

Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.57.0-insider a18ea9c9ec25c59e599ba3c2eb668315546dd269 x64
- OS Version: Windows 10

Steps to Reproduce:

1.
2.

**Comment by alexr00** [comment]
@wclr it looks like your insiders version is 7 days old. I just fixed a bug with auto tasks yesterday. Could you install the latest insiders and see if the issue still happens?

**Comment by wclr** [comment]
Hah, right, it works after exiting and re-launch. 

I actually when I noticed a bug a few days ago decided to wait for the update, because definitely, it could not pass unnoticed. But today it seemed to me that vscode has updated, and the bug was still there I decided to create the issue. But after exiting and launching again it started the updated version. 

I just have the side activity bar hidden and don't see the notification that vscode can be updated, maybe this notification could be shown also in the bottom status bar.

So I believe may close it, thanks for fixing the bug.
